website/community/Team.md

```diff
@@ -3,7 +3,7 @@ id: team
 title: Team
 ---
 
-import Card from '@site/src/theme/card.jsx';
+import Card from '@site/src/theme/card.tsx';
```

Comment:
We should add the `Card` component to `website/src/theme/MDXComponents.js` and remove all imports.